packages/client/hmi-client/src/components/dataset/tera-dataset-description.vue

```diff
@@ -47,7 +47,7 @@
 				<div v-for="(value, key) in dataset.metadata" :key="key" class="row">
 					<template v-if="key.toString() !== 'preview'">
 						<div class="col key">
-							{{ snakeToCapitalized(key) }}
+							{{ snakeToCapitalized(key.toString()) }}
```

Comment:
Seems unrelated, just wanted to make sure this is intentional

Reply:
Not intentional - I started out working with `utils/text.ts` which is where the function came from, but changed my mind and moved to a separate dsv-util, I thought I would just leave existing changes in since they are just type-cleanups.